src/components/Navbar.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+    <nav class="navbar">
+        <div class="logo">
+            <img :src="keyclashlogo" alt="KeyClash Logo">
+        </div>
+        <ul class="nav-links">
+            <li><a href="/">Home</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#about-us">About us</a></li>
+        </ul>
+    </nav>
+</template>
+
+<script>
+import keyclashlogo from '@/public/keyclashlogo.png'
```

Comment:
What does the @ mean?

Reply:
The "@" is something really nice its used in vue as a shorthand for v-on. But in this case the "@" references the src folder, this isnt a vue feature, its a feature from build tools (vue-cli, vite).